lnbits/app.py

```diff
@@ -228,9 +230,7 @@ def check_installed_extension_files(ext: InstallableExtension) -> bool:
     if ext.has_installed_version:
         return True
 
-    zip_files = glob.glob(
-        os.path.join(settings.lnbits_data_folder, "extensions", "*.zip")
-    )
+    zip_files = glob.glob(os.path.join(settings.lnbits_data_folder, "zips", "*.zip"))
```

Comment:
shoudln't this be `lnbits_extensions_path`?

```suggestion
    zip_files = glob.glob(os.path.join(settings.lnbits_extensions_path, "zips", "*.zip"))
```

Reply:
The idea was that:
  - `settings.lnbits_extensions_path` - is for code
  - `settings.lnbits_data_folder` - is for random files, SQLite files, and (now) zips